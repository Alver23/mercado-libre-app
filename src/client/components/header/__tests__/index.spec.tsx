// Dependencies
import { render, screen, act } from '@testing-library/react';

// Under test file
import Header from '../index';

describe('<Header />', () => {
  it('should render correctly', () => {
    render(<Header />);
    act(() => {
      const linkElement = screen.getByText('Home');
      expect(linkElement).toBeInTheDocument();
    });
  });
});
