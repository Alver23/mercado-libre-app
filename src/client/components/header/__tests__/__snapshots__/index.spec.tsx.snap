// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should save a snapshot of the component 1`] = `
<header
  className="h-24 bg-yellow-600 flex justify-center items-center"
>
  <nav
    className="container mx-auto flex justify-center items-center"
  >
    <div
      className="w-full lg:w-10/12"
    >
      <div
        className="flex flex-row"
      >
        <Link
          href="/"
        >
          <a
            className="mr-12 flex justify-center items-center"
          >
            <Image
              alt="Mercado libre"
              height={36}
              src="/assets/logo.png"
              width={54}
            />
          </a>
        </Link>
        <div
          className="w-full"
        />
      </div>
    </div>
  </nav>
</header>
`;
